cmd/root.go

```diff
@@ -527,6 +527,9 @@ CPU may be throttled and a background refresh cannot run reliably
 against all specified instances. If an instance is unreachable, the Proxy exits with a failure
 status code.`)
 
+	localFlags.BoolVar(&c.conf.IgnoreFailedConnections, "ignore-failed-connections", false,
+		`If set, the Proxy will not exit on startup when Cloud SQL instances are unreachable.`)
```

Comment:
Let's phrase this in the positive:

"If set, the Proxy will skip any instances that cannot be found" or similar.